Makefile

```diff
@@ -0,0 +1,44 @@
+AUTOSCALER_SCHEDULER_IMG ?= kube-autoscale-scheduler:dev
+AUTOSCALER_AGENT_IMG ?= autoscaler-agent:dev
+VM_INFORMANT_IMG ?= vm-informant:dev
+EXAMPLE_VM_IMG ?= vm-example:dev
+
+.PHONY: docker-build
+docker-build:
+	docker buildx build \
+		--tag $(AUTOSCALER_SCHEDULER_IMG) \
+		--build-arg "GIT_INFO=n/a" \
+		--file build/autoscale-scheduler/Dockerfile \
+		.
+	docker buildx build \
+		--tag $(AUTOSCALER_AGENT_IMG) \
+		--build-arg "GIT_INFO=n/a" \
+		--file build/autoscaler-agent/Dockerfile \
+		.
+	docker buildx build  \
+		--tag $(VM_INFORMANT_IMG) \
+		--build-arg "GIT_INFO=n/a" \
+		--file build/vm-informant/Dockerfile \
+		.
+
+.PHONY: kind-load
+kind-load: docker-build
+	kind load docker-image $(AUTOSCALER_SCHEDULER_IMG)
+	kind load docker-image $(AUTOSCALER_AGENT_IMG)
+	kind load docker-image $(VM_INFORMANT_IMG)
```

Comment:
```suggestion
```

Don't need to load the VM informant image. The image is only created to store the VM informant binary before building the test VM.